platform-sdk/swirlds-config-processor/src/main/java/com/swirlds/config/processor/antlr/generated/JavaLexer.java

```diff
@@ -0,0 +1,1368 @@
+/*
```

Comment:
Is this necessary to check in generated files?

Reply:
otherwise we need to generate the files for each build. Since the grammatic of Java won't change that often we decided to commit it